### ISSUE TITLE
Use github flow

### ISSUE THREAD

**Issue by mteodori** [Activiti/Activiti#3088]
At the moment we have a mix of github flow on the repositories that are in charge of producing docker images and git flow on most of the others while still using just the develop branch and forgetting to keep  master uptodate when we release.
In order to avoid confusion we should just use master with github flow and delete develop and match the same behaviour of the repositories with docker images.
All the updatebot/Jenkinsfile configurations should be updated to match as noted by @almerico

**Comment by mteodori** [comment]
we use now gitflow on:
* https://github.com/Activiti/Activiti
* https://github.com/Activiti/activiti-cloud
* https://github.com/Activiti/activiti-cloud-application

**Comment by erdemedeiros** [comment]
@mteodori is this still needed?

**Comment by mteodori** [comment]
yes for later, I think we can have just develop and rename to main
